enclave-manager/web/src/components/DownloadButton.tsx

```diff
@@ -17,9 +18,20 @@ export const DownloadButton = <IsIconButton extends boolean>({
   isIconButton,
   ...buttonProps
 }: DownloadButtonProps<IsIconButton>) => {
-  const handleDownloadClick = () => {
+  const handleDownloadClick = async () => {
     if (isDefined(valueToDownload)) {
       const v = typeof valueToDownload === "string" ? valueToDownload : valueToDownload();
+
+      if (isAsyncIterable(v)) {
+        const writableStream = streamsaver.createWriteStream(fileName);
```

Comment:
@Dartoxian I think there's a good chance this may work fine, but I wonder if you've tried downloading from your cloud instance? It's possible this might not work going over the proxies (without some tweaking) but it's possible it will.

Reply:
Yep I've verified that the `streamsaver` library works in the cloud as it is also used to download artifact tgz files in the cloud.